applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -407,6 +425,13 @@ The ``<cmd>`` command is a string, and can be used as follows:
 
   For details, see the :kconfig:option:`CONFIG_LWM2M_CARRIER_PDN_TYPE` Kconfig option.
 
+* ``AT#XCARRIERCFG="queue_mode"[,<0|1>]``
+
+  Configure whether the LwM2M carrier library is to inform the server that it may be disconnected for an extended period of time.
+  This configuration corresponds to the Queue Mode Operation as defined in the OMA LwM2M specification.
```

Comment:
```suggestion
  This configuration corresponds to the Queue Mode operation as defined in the OMA LwM2M specification.
```

Reply:
Updated locally. Will add next time I push, but would like to run CI first, and hopefully merge PR to nrfxlib.

Reply:
I'm ok with this suggestion, however the LwM2M specification seems to capitalise all three terms
![Uploading image.png…]()

Reply:
I missed to check the spec. thanks.